.gitignore

```diff
@@ -3,3 +3,6 @@ Gemfile.lock
 **/.DS_Store
 .idea/
 *.iml
+
+# Generated test files
+collectd.json
```

Comment:
One test case writes the json output to this file on line spec/classes/init_spec.rb#L325, not sure why the File.write is necessary there as there are no other test cases that use it, would either use a tempfile or include it in the ignore